template/CHART_NAME/values.yaml

```diff
@@ -445,6 +497,55 @@ ingress:
   ##
   secrets: []
 
```

Comment:
Be careful, I think `persistence` would fall under the "Traffic Exposure" section.

Reply:
Not sure what you mean

Reply:
The values metadata divides the values.yaml using `@section`.

Currently, the `persistence.*` values are located under `## @section Traffic Exposure Parameters`
We usually have them under their own section `@section Persistence parameters`